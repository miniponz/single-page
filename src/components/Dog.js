import React from 'react';

export default function Dog() {
  const dog = {
    name: 'Ana Dogg',
    age: 2,
    weight: '28 lbs'
  };

  return (
    <dl>
      <dt>Name</dt>
      <dd>{dog.name}</dd>

      <dt>Age</dt>
      <dd>{dog.age}</dd>

      <dt>Weight</dt>
      <dd>{dog.weight}</dd>
    </dl>
  );
}
