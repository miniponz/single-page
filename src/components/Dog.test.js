import React from 'react';
import { shallow } from 'enzyme';
import Dog from './Dog';

describe('Dog Compnent', () => {
  it('matches the dog thing', () => {
    const wrapper = shallow(<Dog />);

    expect(wrapper).toMatchSnapshot();
  });
});
