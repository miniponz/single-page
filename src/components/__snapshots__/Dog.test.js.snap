// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dog Compnent matches the dog thing 1`] = `
<dl>
  <dt>
    Name
  </dt>
  <dd>
    Ana Dogg
  </dd>
  <dt>
    Age
  </dt>
  <dd>
    2
  </dd>
  <dt>
    Weight
  </dt>
  <dd>
    28 lbs
  </dd>
</dl>
`;
